src/sinks/splunk_hec/logs/config.rs

```diff
@@ -106,9 +128,11 @@ pub struct HecLogsSinkConfig {
     ///
     /// [global_timestamp_key]: https://vector.dev/docs/reference/configuration/global-options/#log_schema.timestamp_key
     #[serde(default = "crate::sinks::splunk_hec::common::timestamp_key")]
+    #[configurable(metadata(docs::examples = "timestamp", docs::examples = ""))]
```

Comment:
What's the behavior here if `""` is passed in?

Reply:
Have added a note.